index.html

```diff
@@ -71,12 +71,39 @@ <h2>I am a Software Developer</h2>
 		<div class="text1">
 			<h2>My Recent Works <img src="images/line.png" alt=""></h2>
 		</div>
+
+		<div class="popupcontainer" id="popup">
+			<div class="popupcontainer2">
+				<div class="popuptitle">
+					<h1 class="header2">Multi Post Story</h1>
+					<button class="closebtn active" type="button" onclick="togglePopup()">&times;</button>
+				</div>
+				<div class="langcontainer">
+					<ul class="ulcontainer">
+						<li class="langg">html</li>
+						<li class="langg">bootstrap</li>
+						<li class="langg">Ruby on rails</li>
+					</ul>
+				</div>
+				<div class="popupcontainer3">
+					<img src="images/Snapshoot Portfolio.png" class="image2" alt="">
+					<div class="description1">
+						<p class="paragraph2">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+						<div class="btncontainer">
+							<button class="popbtn" type="button"><a href="https://mikey1-nuel.github.io/Setup-and-Mobile-First/"><img src="images/Enable.png" alt="enable"></a></button>
+							<button class="popbtn" type="button"><a href="https://github.com/Mikey1-nuel/Setup-and-Mobile-First"><img src="images/Enable2.png" alt=""></a></button>
+						</div>
+					</div>
+				</div>
+			</div>
+		</div>
+
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/225256846-a1c2850d-ccb1-4f10-a768-ab6fcca95cf7.png)

- [ ] Your branch name is mobile-menu-branch which doesn't make any sense here because your project is about popup-window so your branch name should be like **feature/add-popup** which is descriptive and professional 👍